crates/iroha_executor/src/default/custom/mod.rs

```diff
@@ -0,0 +1,37 @@
+use iroha_executor_data_model::custom::multisig::MultisigInstructionBox;
+
+use super::*;
+use crate::prelude::{Execute, Vec, Visit};
+
+mod multisig;
+
+pub fn visit_custom_instructions<V: Execute + Visit + ?Sized>(
+    executor: &mut V,
+    isi: &CustomInstruction,
+) {
+    if let Ok(isi) = MultisigInstructionBox::try_from(isi.payload()) {
+        return isi.visit_execute(executor);
+    };
+
+    deny!(executor, "unexpected custom instruction");
+}
```

Comment:
This concrete implementation for "default custom" may sound strange, what do you think

Reply:
Planned to be `default::isi::visit_custom_instruction`